python/src/com/jetbrains/python/PyGotoSymbolContributor.java

```diff
@@ -45,6 +42,7 @@
     symbols.addAll(PyModuleNameIndex.getAllKeys(project));
     symbols.addAll(StubIndex.getInstance().getAllKeys(PyFunctionNameIndex.KEY, project));
     symbols.addAll(StubIndex.getInstance().getAllKeys(PyVariableNameIndex.KEY, project));
+    symbols.addAll(StubIndex.getInstance().getAllKeys(PyClassAttributesIndex.KEY, project));
```

Comment:
I'm sort of scared of add instance attributes here for performance reasons in large projects. Can we put a limit on the max results we return?